docs/articles/new-expensify/expensify-card/Dispute-Expensify-Card-transaction.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Dispute Expensify Card transaction
+description: Dispute an unrecognized, unauthorized, or fraudulent charge
+---
+<div id="new-expensify" markdown="1">
+
+When using your Expensify Visa® Commercial Card, you may come across transactions that you want to dispute, including:
+
+- Unrecognized, unauthorized, or fraudulent charges
+   - Charges made with your card after it was lost or stolen
+   - Unauthorized charges while your card is still in your possession
+   - Continued charges for a canceled recurring subscription
+- Service disputes
+   - Damaged or defective merchandise
+   - Charges for merchandise that was never received
+   - Duplicate charges for a single transaction
+   - Transactions of an incorrect amount
+   - Refund not received after a return
+
+# Dispute a transaction
+
+If you spot a transaction error on your Expensify Card, 
+
+1. Contact the merchant. They can often address the issue promptly.
+2. If you are unable to resolve the issue with the merchant, contact us immediately by opening your chat with Expensify Concierge in your Expensify Chat inbox, or by emailing concierge@expensify.com to start the dispute process. Provide the following information:
+   - Details about the disputed charge, including why you’re disputing it, what occurred, and any steps you’ve taken to address the issue
+   - Supporting documentation like receipts or cancellation confirmations
```

Comment:
This is also a little weird on spacing cc @rushatgabhane - more space between 2 and 3 than between 1 and 2

<img width="775" alt="image" src="https://github.com/Expensify/App/assets/5487802/68f9750b-d3ba-4ee7-894a-9bc60f8f28a9">

Reply:
Made this - https://github.com/Expensify/App/issues/44107 - for the future, if you want to take @rushatgabhane ? Or we can open it up.